x/qgb/handler.go

```diff
@@ -9,7 +9,8 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-// NewHandler uses the provided qgb keeper to create an sdk.Handler
+// Can be deleted after implementing the Orchestrator and Relayer as per QGB ADR-005.
+// NewHandler uses the provided qgb keeper to create an sdk.Handler.
```

Comment:
[question]

just double checking, do we have an issue for this?

Reply:
we have this: https://github.com/celestiaorg/celestia-app/issues/684 
Should be detailed more, but I guess it summarizes everything until we create a more concrete tracking issue with explicit tasks